### PR TITLE
[KAVYA-20] -> [development] Done with the functionality of display, edit and update  project and emp...

### PR DESCRIPTION
**Pull request by Kavyahs**
...loyee details.

### CODE REVIEW

File: app/controllers/employees_controller.rb [hunk 1]
```diff
@@ -34,6 +34,9 @@ def update
     @employee = Employee.find(params[:id])
     if @employee.valid?
       @employee.update(employee_params)
+      if params["Employprojects"].present?
+        @employee.saveprojects(params["Employprojects"]["p"], @employee.id)
```
Comment (kedarnag138) [inline app/controllers/employees_controller.rb:38]: What are you trying to achieve here ?

File: app/controllers/projects_controller.rb [hunk 2]
```diff
@@ -17,6 +22,27 @@ def create
...
+    respond_to do |format|
+      format.js {}
+    end
+  end
+
+  def update
+    @project = Project.find(params[:id])
+    if @project.valid?
+      @project.update(project_params)
+      if params["Projectemployee"].present?
```
Comment (kedarnag138) [inline app/controllers/projects_controller.rb:37]: What are you doing here as well ?

File: app/controllers/employees_controller.rb (hunk 1 above)
Comment (Kavyahs) [inline app/controllers/employees_controller.rb:38]: In the Update action of employees_controller.rb, I am updating the changes of employee details and saving newly selected projects. Here 'saveprojects' is a method which I defined in Employee model 'employee.rb'.

File: app/controllers/projects_controller.rb (hunk 2 above)
Comment (Kavyahs) [inline app/controllers/projects_controller.rb:37]: Here also I am updating project details and saving newly selected employees for that project.

For edit action I am sending Js request on click of project card. To display respective project details in edit for I need @project = Project.find(params[:id]). And I have to display employees name list in 'Employees' field of the same form so I need @employees = Employee.all.

**Pull request merged**